src/descriptor/mod.rs

```diff
@@ -506,13 +547,17 @@ impl<Pk: MiniscriptKey + ToPublicKey> Descriptor<Pk> {
             }
             Descriptor::Sh(ref d) => d.encode(),
             Descriptor::Wsh(ref d) | Descriptor::ShWsh(ref d) => d.encode(),
+            Descriptor::ShSortedMulti(ref smv) => smv.encode(),
+            Descriptor::WshSortedMulti(ref smv) | Descriptor::ShWshSortedMulti(ref smv) => {
+                smv.encode()
+            }
         }
     }
 
     /// Returns satisfying witness and scriptSig to spend an
     /// output controlled by the given descriptor if it possible to
     /// construct one using the satisfier.
-    pub fn get_satisfication<S: Satisfier<Pk>>(
+    pub fn get_satisfication<S: Satisfier<Pk> + Satisfier<bitcoin::PublicKey>>(
```

Comment:
This should not be required. See the comments below for clarification.

Reply:
Done